project/scripts/is-version-published.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Check whether a specific version of the Scala 3 compiler is published to Maven Central
+#
+# Usage:
+#   is-version-published.sh <version_string>
+# e.g.
+#   ./is-version-published.sh 3.0.1-RC1-bin-20210411-b44cafa-NIGHTLY
+#
+# Exit status:
+#   zero      if the specified version is published on Maven Central
+#   non-zero  otherwise
+
+ver=$1
+if [[ -z "$ver" ]]; then
+  echo "error: missing version parameter"
+  echo "usage: $0 <version_string>"
+  exit 2
+fi
+
+set -eu
+
+major=${ver%%-bin-*}
```

Comment:
Once https://github.com/lampepfl/dotty/pull/11952 is merged, the binary version will be fixed to 3, so this won't work anymore, I suggest waiting until that PR is merged then just using _3.

Reply:
https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/ is up, so you should be able to change the script to cut the version number after the first `.` or hardcode 3.